appveyor.yml

```diff
@@ -51,7 +51,7 @@ install:
 
   # Upgrade to the latest version of pip to avoid it displaying warnings
   # about it being out of date.
-  - pip install --disable-pip-version-check --user --upgrade pip wheel
+  - pip install --disable-pip-version-check --user --upgrade pip~=9.0 wheel
```

Comment:
What's the pip pin fixing? There should at least be a comment probably?

Also, I'm pretty sure you're supposed to run `python -m pip` on Windows when upgrading pip (because otherwise it tries to upgrade the same `pip.exe` that it's currently running and that makes Windows very cranky). Possibly that could be the original issue?